scripts/aggregate-junit-xml.js

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
Small tweaks to make the script run successfully on Windows even when there's some execution failures.